src/pages/workspace/WorkspaceMoreFeaturesPage.tsx

```diff
@@ -316,6 +320,35 @@ function WorkspaceMoreFeaturesPage({policy, route}: WorkspaceMoreFeaturesPagePro
         },
     ];
 
+    if (isBetaEnabled(CONST.BETAS.UBER_FOR_BUSINESS)) {
+        integrateItems.push({
+            icon: Illustrations.ReceiptPartners,
+            titleTranslationKey: 'workspace.moreFeatures.receiptPartners.title',
+            subtitleTranslationKey: 'workspace.moreFeatures.receiptPartners.subtitle',
+            isActive: policy?.areReceiptPartnersEnabled ?? false,
+            pendingAction: policy?.pendingFields?.areReceiptPartnersEnabled,
+            disabledAction: () => {
+                // TODO: When Uber integration is added, check if any integration exists
+                // and show the warning modal to disconnect integration first
+                setIsReceiptPartnersWarningModalOpen(true);
+            },
+            action: (isEnabled: boolean) => {
+                if (!policyID) {
```

Comment:
How is this possible?

Reply:
Not sure I follow. Are you saying this can never be falsy but we are checking everywhere anyway?
And if so, why would we do that?